src/index.js

```diff
@@ -6,17 +6,14 @@
 module.exports = {
   env: {
     es6: true,
-    jasmine: true,
     jest: true,
-    mocha: true,
```

Comment:
I'm not sure about removing support for mocha and jasmine, at least in this PR...